cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -34,3 +34,5 @@
 default['cluster']['fsx_volume_junction_paths'] = ''
 default['cluster']['raid_type'] = ''
 default['cluster']['raid_vol_ids'] = ''
+default['cluster']['scheduler_slots'] = 'vcpus'
+default['cluster']['ephemeral_dir'] = '/scratch'
```

Comment:
nit: it is better to keep the "storage related" attributes close to each other

Reply:
Sure.